src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -30,44 +30,77 @@
                 </div>
             </PageTitleSection>
             <ToolbarSection>
-                <div slot="" class="trace-header">
-                    <FluentOverflow Class="trace-header-details">
-                        <ChildContent>
-                            <FluentOverflowItem>
-                                @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTraceStartHeader)] <strong title="@FormatHelpers.FormatDateTime(TimeProvider, _trace.FirstSpan.StartTime, MillisecondsDisplay.Full)">@FormatHelpers.FormatDateTime(TimeProvider, _trace.FirstSpan.StartTime, MillisecondsDisplay.Truncated)</strong>
-                            </FluentOverflowItem>
-                            <FluentOverflowItem>
-                                @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailDurationHeader)] <strong>@DurationFormatter.FormatDuration(trace.Duration)</strong>
-                            </FluentOverflowItem>
-                            <FluentOverflowItem>
-                                @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailResourcesHeader)] <strong>@_resourceCount</strong>
-                            </FluentOverflowItem>
-                            <FluentOverflowItem>
-                                @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailDepthHeader)] <strong>@_maxDepth</strong>
-                            </FluentOverflowItem>
-                            <FluentOverflowItem>
-                                @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTotalSpansHeader)] <strong>@trace.Spans.Count</strong>
-                            </FluentOverflowItem>
-                        </ChildContent>
-                        <OverflowTemplate Context="overflow">
-                            <FluentTooltip UseTooltipService="false" Anchor="@overflow.IdMoreButton" Class="trace-header-tooltip">
-                                @foreach (var item in overflow.ItemsOverflow)
-                                {
-                                    <div style="margin: 5px;">@item.ChildContent</div>
-                                }
-                            </FluentTooltip>
-                        </OverflowTemplate>
-                    </FluentOverflow>
-                    <div class="trace-header-filters">
+                @if (ViewportInformation.IsDesktop)
+                {
+                    <div slot="" class="trace-header">
```

Comment:
I'd rather this trace header also be visible on the page in mobile view. Even though it is displayed in the header, and we typically don't display header information, this summary of the trace is important don't.

Don't display it in the mobile filter popup

Reply:
The overflow should prevent it from taking up a lot of room on small screens. It will be one like max.

Reply:
> The overflow should prevent it from taking up a lot of room on small screens. It will be one like max.

That's the problem. It looks really bad
<img width="358" height="611" alt="image" src="https://github.com/user-attachments/assets/0d4386fb-7904-40eb-bac8-2febc25a09ef" />

This is why I added the unordered list below. I updated to this, what do you think?

<img width="727" height="1233" alt="image" src="https://github.com/user-attachments/assets/f67049cc-fd7a-4a70-aa44-e29280dbd58f" />

Reply:
No, I mean display the details on the main traces details view in mobile, not the tool bar view.

Like this:

<img width="882" height="966" alt="image" src="https://github.com/user-attachments/assets/3ad2d05e-0b54-4c24-853d-326f498c67e3" />

I made changes for this.